### ISSUE TITLE
Remove "Underline" from CKEditor default config

### ISSUE THREAD

**Issue by smillart** [openfed/openfed#23]
A lot of users still often underline text to create a sense of urgency or importance. However, underlined text on a page looks like a link so users may try to select it with no success.

**Underlined text is also not semantic meaning** it really doesn’t tell the AT users anything. If they are listening to the page with a screen reader, there is no difference between plain text and underlined text. Since it looks like a link and doesn’t mean anything, underlining text often has the opposite effect of what you’re trying to accomplish.

**Bottom line:** Reserve underlining for identifying links. **Do not underline text!**
